.github/workflows/deploy-pr-to-s3.yml

```diff
@@ -0,0 +1,47 @@
+name: Deploy the pull request to AWS S3
+on: [pull_request]
+
+env:
+  PR_NUMBER: ${{ github.event.pull_request.number }}
+  REPO_NAME: ${{ github.event.repository.name }}
+
+jobs:
+  deploy-to-s3:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+        with:
+          persist-credentials: false
+
+      - name: Read .nvmrc
+        run: echo "::set-output name=NODE_VERSION::$(cat .nvmrc)"
+        id: nvm
+
+      - uses: actions/setup-node@v1
+        with:
+          node-version: ${{ steps.nvm.outputs.NODE_VERSION }}
+
+      - name: Install dependencies
+        run: npm ci
+
+      - name: Build
+        run: npm run build:storybook -- -o ./dist/${{ env.REPO_NAME }}/${{ env.PR_NUMBER }}
+
+      - name: Deploy
+        uses: jakejarvis/s3-sync-action@master
+        with:
+          args: --acl public-read
+        env:
+          AWS_S3_BUCKET: ${{ secrets.AWS_S3_PR_DEPLOYMENT_BUCKET }}
+          AWS_ACCESS_KEY_ID: ${{ secrets.AWS_ACCESS_KEY_ID }}
+          AWS_SECRET_ACCESS_KEY: ${{ secrets.AWS_SECRET_ACCESS_KEY }}
+          SOURCE_DIR: 'dist'
+          DEST_DIR: '${{ env.REPO_NAME }}/${{ env.PR_NUMBER }}'
+
+      - name: Add comment
+        uses: marocchino/sticky-pull-request-comment@v1
+        with:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+          message: |
+            🚀 This PR has been deployed here: https://frontend-pulls.homeday.de/${{ env.REPO_NAME }}/${{ env.PR_NUMBER }}/
```

Comment:
The trailing slash here is necessary because Storybook uses relative paths:
<img width="425" alt="Screen Shot 2020-10-02 at 5 23 36 PM" src="https://user-images.githubusercontent.com/30146019/94941213-df3fa480-04d4-11eb-8b23-be6c030b02dc.png">
And they don't have a way to specify the base url (like in the Vue apps) unfortunately :/ ... or I just couldn't find it in the docs.